core/output/grunt.js

```diff
@@ -0,0 +1,125 @@
+var shim = require.once('../shim.js'),
+    server = require.once('http').createServer();
+    io = require.once('socket.io')(server),
+    port = 49886,
+    functions = null,
+    api = null,
+    callback = null,
+    socket = null,
+
+    sendMessage = function(message, thread) {
+        socket.emit('message', {
+            content: message,
+            thread_id: thread
+        });
+    },
+
+    sendPrivateMessage = function(message, thread, senderId) {
+        socket.broadcast.emit('privateMessage', {
+            content: message,
+            thread_id: thread,
+            sender_id: senderId
+        });
+    },
+
+    sendUrl = function(url, thread) {
+        socket.broadcast.emit('url', {
+            content: url,
+            thread_id: thread
+        });
+    },
+
+    sendImage = function(type, image, description, thread) {
+        socket.broadcast.emit('image', {
+            type: type,
+            content: image,
+            thread_id: thread,
+            description: description
+        });
+    },
+
+    sendFile = function(type, file, description, thread) {
+        socket.broadcast.emit('file', {
+            type: type,
+            content: file,
+            thread_id: thread,
+            description: description
+        });
+    },
+
+    sendTyping = function(thread) {
+        socket.broadcast.emit('typing', {
+            thread_id: thread
+        });
+    },
+
+    setTitle = function(title, thread) {
+        socket.broadcast.emit('title', {
+            content: title,
+            thread_id: thread
+        });
+    },
+
+    recMessage = function(data) {
+        if (data) {
+            if (!data.thread_id) {
+                data.thread_id = '-1';
+            }
+            if (!data.sender_id) {
+                data.sender_id = '123456789';
+            }
+            if (!data.sender_name) {
+                data.sender_name = 'test user';
+            }
+            if (!data.content) {
+                data.content = '';
+            }
+
+            var event = shim.createEvent(data.thread_id, data.sender_id, data.sender_name, data.content);
+            return callback(api, event);
+        }
+        console.error('Received messsage with no content');
+    };
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/theprojectshop/Kassy/file/2531448095/issues/source?bid=3282987&fileBranchId=3283883#l82)